doc/es_stores.md

```diff
@@ -0,0 +1,71 @@
+```
+
+When `aliased` is set to true, it will not be considered unless one of `rollover` conditions are set, `max_docs` or `max_age`. Once again, this is configurable per store.
+```
+ctia.store.es.default.rollover.max_docs=10000000
+ctia.store.es.default.rollover.max_age=6m
+ctia.store.es.event.rollover.max_docs=10000000
+ctia.store.es.event.rollover.max_age=1m
+ctia.store.es.event.rollover.max_docs=5000000
+ctia.store.es.event.rollover.max_age=1w
+```
+
+## Unaliased stores: 1 store = 1 index
+
+- CTIA directly writes on configured index name
+```
+ctia.store.es.event.indexname=ctia_event
+ctia.store.es.sighting.indexname=ctia_sighting
+ctia.store.es.relationship.indexname=ctia_relationship
+...
+```
+- At the Initialization, for each unaliased ES store, CTIA creates one template with proper index settings and type mapping. The index will be created with first document insertion, and configured from that template.
+- CRUD operations and Search requests directly target the index.
+
+Elasticsearch search indices have a fixed number of shards. It's impossible to modifiy this number of shards without reindexing the corresponding index. This limits horizontal scaling when shards become to big. Unless you are sure about the fact that configured stores will not exceed 20Gb per shard (meaning you can plan that), we strongly discourage to use unaliased stores. Some stores that could remain small are for instance `malware` or `indicator`. You can configure the number of shards as follow:
+``
+ctia.store.es.default.shards=5
+
+## Aliased stores: 1 store / n indices, read/write aliases
+We use aliases and _rollover API to ease horizontal scaling on very big stores: 
+- multiple indices, with read alias on all indices and write alias on most recent index. read alias is the store name, write alias is the store name suffixed by -write, index names are the store name suffixed by a number.
+- At Initialization CTIA generates a template with a read alias and the first index with the write and read aliases.
+- ES CRUD operations do not work on multiple indices:
+    - GET is replaced by a _search request with an ids query on read alias
+    - CREATE are done on the most recent index using write alias
+    - UPDATE and DELETE require 2 steps:
+        - a _search with an ids query on read alias to retrieve the real index of the targeted document.
+        - update / delete request on that retrieved index.
+- _rollover api is used to create a new index and to “roll over” write alias on that new index, when some rules (max_docs and / or max_age) are matched.
+
+
+### Rollover task
+Currently, CTIA uses Elasticsearch 5.6 which requires to manually request _rollover API. Since indices could be shared by multiple CTIA instances, we designed an external task that must be frequently launch to request _rollover with configured conditions. This task requires no parameter, it simply reads the conf and to detects stores and their rollover conditions. 
```

Comment:
```suggestion
Currently, CTIA uses Elasticsearch 5.6 which requires to manually request _rollover API. Since indices could be shared by multiple CTIA instances, we designed an external task that must be launched frequently to request _rollover with configured conditions. This task requires no parameter, it simply reads from configuration, detects stores and their rollover conditions. 
```